### DISCUSSION TITLE
Data Sharing Policy on MIMIC and Demo Datasets

### DISCUSSION THREAD

**Discussion by aegis301** [MIT-LCP/mimic-code#1626] [MIMIC-General] [answered]
Dear MIMIC-Maintainers,

me and my team are currently developing open source toolboxes to work with ICU time series data. One aspect of this toolbox will be different scoring systems. We want to evaluate some of these scoring tools against human level performance. The goal is to publish one or more papers about these tools in a scientific journal to achieve high trustability. 

Given the high quality of MIMIC data, it is a valuable resource for our testing population. In order to do so, we want to show MIMIC data to trained physicians that should generate human level labels and then evaluate them against our labels. Other than the developers working on the toolbox, these physicians however have no prior experience with working with MIMIC and are therefore not yet CITI-certified or MIMIC-accredited. We therefore plan to use the publicly available MIMIC Demo Dataset in order to show this data to the physicians. E.g. SAPSII scores would be calculated on this pipeline. Concerning this approach we have two questions:

1. Is it permitted to show MIMIC Demo data to non-accredited physicians or do physicians seeing this data have to be accredited via Physionet?
2. Would it be possible for us to publish the data we used along with the labels we generated, e.g. in a Supplements File? This would vastly improve our planned publication and give developers and researchers the chance to evaluate the performance themselves. It would offer us the opportunity to be as open about the performance of our models as possible.

Trust and openness in our development are of highest priority for us. However, data privacy concerns and MIMIC publishing guidelines are of course in the center of our focus as well. We want to make absolutely sure we are working within the appropriate limits and according to your regulations.

Thank you so much for your hard work and for the high quality of resources you provide to us and other researcher teams. We highly appreciate it and hope to be able to contribute to this community with our work.

Kind regards

**Comment by tompollard** [comment, marked as answer]
Hi @aegis301, thanks for checking!

> Is it permitted to show MIMIC Demo data to non-accredited physicians or do physicians seeing this data have to be accredited via Physionet?

Almost all of our demo datasets are released under permissive, open data licenses, allowing you to reshare without restriction. e.g. the MIMIC-IV Clinical Dataset https://physionet.org/content/mimic-iv-demo/ is published under the [Open Data Commons Open Database License v1.0](https://physionet.org/content/mimic-iv-demo/view-license/2.2/).

<img width="751" alt="Screenshot 2023-08-16 at 9 39 56 AM" src="https://github.com/MIT-LCP/mimic-code/assets/822601/7296b956-ef2d-4089-8833-cee2d265c27a">

> Would it be possible for us to publish the data we used along with the labels we generated, e.g. in a Supplements File?

The open data license is permissive so this is acceptable for the demo dataset. In general, I think we would prefer instead that you pointed people to PhysioNet where possible though, so that they can download from source:

> Johnson, A., Bulgarelli, L., Pollard, T., Horng, S., Celi, L. A., & Mark, R. (2023). MIMIC-IV Clinical Database Demo (version 2.2). PhysioNet. https://doi.org/10.13026/dp1f-ex47.

PhysioNet is an archival repository, so we are committed to ensuring that the data will not change. Your labels will continue to be linkable to this version of the demo.
